pages/linux/cewl.md

```diff
@@ -0,0 +1,24 @@
+# cewl
+
+> URL spidering tool for making a cracking wordlist from web content.
```

Comment:
I don't understand what is a cracking wordlist. Can you explain ?

Reply:
/cc @dhondta

Reply:
@agnivade See https://en.wikipedia.org/wiki/Password_cracking
@sbrl What do you mean with `/cc` ?

Reply:
/cc like [CC in email](https://en.wikipedia.org/wiki/Carbon_copy#Email). Just a fancy way to mention you. /ping @dhondta is another fancy way.

Reply:
Thanks for the clarification. I initially thought it was too difficult for a beginner to understand, but now looking at it again, I think it's easy enough.